src/main/webapp/js/adminSearch.es6

```diff
@@ -53,6 +53,57 @@ $(document).ready(() => {
     $('.resetGoogleIdButton').click((e) => {
         e.stopPropagation();
     });
+
+    $('.openMailEditorWithDefaults').submit(function (e) {
+        e.preventDefault();
+        const receiveremailId = $('input[name=studentEmail]').val();
+        const subjectType = $(this).find('input.subjectType').val();
+        const studentName = $(this).find('input.studentName').val();
+        const courseName = $(this).find('input.courseName').val();
+        const Id = $(this).find('input.Id').val();
```

Comment:
I think using different constants for `courseId` and `fsName` would be better. `Id` may be too generic and not descriptive of either value.